modules/bootkube/variables.tf

```diff
@@ -160,3 +160,9 @@ variable "pull_secret" {
   type        = "string"
   description = "Your pull secret.  Obtain this from your Tectonic Account: https://account.coreos.com."
 }
+
+variable "worker_ign_config" {
+  description = "Worker ignition config"
```

Comment:
Is this temporary?  I'd have expected the machine API operator to manage worker ignition configs on its own.

Reply:
> If anything, I would expect the machine-config-operator (this is getting confusing...) to manage the ignition configs and provide them in some way to the machine-api-operator.

That makes sense to me.  @abhinavdahiya?